test/NCI.OCPL.Api.Glossary.Tests/Tests/Controllers/TermsControllerTest.cs

```diff
@@ -431,7 +431,7 @@ public async void GetById_WithFallback_TermsPatient()
             .Returns(Task.FromResult(glossaryTerm));
 
             TermsController controller = new TermsController(NullLogger<TermsController>.Instance, termQueryService.Object);
-            GlossaryTerm gsTerm = await controller.GetById("NotSet", AudienceType.Patient, "en", 44771, true);
+            GlossaryTerm gsTerm = await controller.GetById("notset", AudienceType.Patient, "en", 44771, true);
```

Comment:
Again, not a _bad_ change, just not  a necessary one.

Reply:
Same answer as above!